cpp/data/email_watcher.conf

```diff
@@ -0,0 +1,20 @@
+# Each section must provide the following entries:
+# from_host           -- where the emails are originating
+# subject_patttern    -- a PCRE regex that must match the emails' subjects
+# email_body_pattern  -- a PCRE that must match the emails' bodies
```

Comment:
it might be useful to not only have patterns that must match, but also have patterns that must not match.

E.g. if we have a message with "pipeline completed successfully" which also contains a warning at the same time, we might want to forward that message anyway.